### ISSUE TITLE
Height of door control elements

### ISSUE THREAD

**Issue by DWillDB** [NeTEx-CEN/NeTEx#541]
Hello everyone,

We need the height of the door control elements and we believe it should be included in the "EntranceUsageGroup." Is it possible to add the following code to the XSD file? Or would it be possible to submit our own pull request? What would be the best process for this?

https://github.com/NeTEx-CEN/NeTEx/blob/c6c34850192a05e88079d112d340da644f35fe90/xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd#L1195
`<xsd:element name="DoorControlElementHeight" type="LengthType" minOccurs="0">
    <xsd:annotation>
	    <xsd:documentation>Indicates door control element height (in centimeter)</xsd:documentation>
    </xsd:annotation>
</xsd:element>`

Best regards,
David

**Comment by skinkie** [comment]
@davidr1234 try use with a pool request for the EPIAP branch.

so, first fork our branch to your own repository
```
git checkout EPIAP -b enhance_doorcontrolelementheight
do change
git commit -a -m "Fix #541"
git push --set-upstream origin enhance_doorcontrolelementheight
```
...and then do a pull request here (and don't forget to select EPIAP on the left)

**Comment by DWillDB** [comment]
Yes, that is what I thought. But I have no permissions to create a branch.

**Comment by skinkie** [comment]
> Yes, that is what I thought. But I have no permissions to create a branch.

The idea is not to create it on this repository. But fork it into your personal space on github.

![image](https://github.com/NeTEx-CEN/NeTEx/assets/502394/71b8c09f-d923-4eb7-aee8-3a6cc7456c54)

**Comment by nick-knowles** [comment]
DoorControlElementHeight added to EntranceEquipment in UML

**Comment by ue71603** [comment]
@trurlurl  also this one? covered by the PR? Can we close the issue?

**Comment by trurlurl** [comment]
> @trurlurl also this one? covered by the PR? Can we close the issue?

Solved by #541.

**Comment by Aurige** [comment]
Part 1 already updated by @trurlurl
